Makefile

```diff
@@ -206,7 +206,11 @@ $(VENDOR_DIR): glide.lock glide.yaml
 deps: $(VENDOR_DIR)
 
 app/controllers.go: $(DESIGNS) $(GOAGEN_BIN) $(VENDOR_DIR)
-	$(GOAGEN_BIN) bootstrap -d ${PACKAGE_NAME}/${DESIGN_DIR}
+	$(GOAGEN_BIN) app -d ${PACKAGE_NAME}/${DESIGN_DIR}
+	$(GOAGEN_BIN) controller -d ${PACKAGE_NAME}/${DESIGN_DIR} -o controllers/ --pkg controllers --app-pkg app
```

Comment:
@kwk Well! that is a golang discussion. Generally they say to don't use plurals (I agree) but most of golang repositories have plurals. In our case, we have but if you look other important golang repos, you also see them. 

No strong feelings about it :-/